lightning/src/ln/features.rs

```diff
@@ -387,6 +391,17 @@ impl InitFeatures {
 	}
 }
 
+/// Calculates the base32 encoded size of a byte slice
+fn bytes_size_to_base32_size(byte_size: usize) -> usize {
```

Comment:
Can we use the provided implementation for `[u8]` instead of copying this function https://docs.rs/bech32/0.8.0/bech32/trait.Base32Len.html#impl-Base32Len

Reply:
That didn't work sadly... Neither did the original copied&pasted function. I did something super inefficient :grimacing: still need to think through a better way